tests/php/PHPStanBearTest.py

```diff
@@ -0,0 +1,36 @@
+import os
+from queue import Queue
+from shutil import which
+from unittest.case import skipIf
+
+from bears.php.PHPStanBear import PHPStanBear
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.settings.Section import Section
+
+
+@skipIf(which('phpstan') is None, 'PHPStan is not installed')
+class PHPStanBearTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.section = Section('test section')
+        self.uut = PHPStanBear(self.section, Queue())
+        self.test_file1 = os.path.join(os.path.dirname(__file__),
+                                       'test_files',
+                                       'phplint_test1.php')
+        self.test_file2 = os.path.join(os.path.dirname(__file__),
+                                       'test_files',
+                                       'phplint_test2.php')
+
+    def test_run(self):
+        # Test a file with errors and warnings
+        self.check_validity(
```

Comment:
Also, not necessary, but you can use `verify_local_bear` to create good_files and bad_files (good_file is a file which your bear considers as non-style-violating and a bad_file is one which has at least one error/warning/info)